src/videojs-hls.js

```diff
@@ -450,6 +446,8 @@ videojs.Hls.prototype.setCurrentTime = function(currentTime) {
     return currentTime;
   }
 
+  this.lastSegmentLoaded_ = null;
```

Comment:
When we are seeking, we shouldn't care if we load the same segment as last time - this was mostly a problem in flash because we don't have a buffer on seek so we potentially need to load the exact same segment again in this one case.